phpBB/phpbb/language/language.php

```diff
@@ -292,9 +292,10 @@ public function lang_raw($key)
 	 *
 	 * @return string
 	 */
-	public function lang_array($key, $args = array())
+	public function lang_array($key, $args = [])
 	{
 		$lang = $this->lang_raw($key);
+		$args = (array) $args;
```

Comment:
What if we remove these lines instead?

https://github.com/phpbb/phpbb/blob/32932eadc4586356288c37ce4d37481fc417ad0d/phpBB/phpbb/language/language.php#L308-L311

Reply:
It won't return a correct language entry then I guess.

Reply:
I am just assuming that the first condition rules it all?

Reply:
1st condition is on the different var ;)

Reply:
Then `if (empty($args))` due to 3.2.x, otherwise there are more options.

Reply:
It's not needed if we cast `$args` to array for any of cases.

Reply:
Well I agree but seems a bit redundant to me. Anyway.